tidy3d/components/source/freq_range.py

```diff
@@ -159,20 +159,38 @@ def freqs(self, num_points: int) -> NDArray[np.float64]:
         ----------
         num_points : int
             Number of frequency points in a frequency range of interest.
+        spacing: str
+            Mode of frequency sampling.
+            If spacing is set to ``uniform_freq``, frequencies are sampled uniformly over the frequency interval.
+            If set to ``uniform_wvl``, frequencies correspond to uniformly sampled wavelengths over the wavelength interval.
+            Frequencies are sorted in ascending order.
 
         Returns
         -------
         np.ndarray
             a numpy array of uniformly distributed frequency samples in a frequency range of interest.
         """
+        if spacing == "uniform_wvl":
+            return (
+                np.array([td_const.C_0 / self.lda0])
+                if num_points == 1
+                else np.sort(td_const.C_0 / self.ldas(num_points=num_points, spacing=spacing))
```

Comment:
we know that wavelength will be in reverse order here, so no need to call sort. Instead we can do
```
td_const.C_0 / self.ldas(num_points=num_points, spacing=spacing)[::-1]